src/Entity/Course.php

```diff
@@ -15,181 +16,167 @@ class Course
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @var int
      */
     private $id;
 
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string")
+     * @var string
      */
     private $title;
 
     /**
      * @ORM\Column(type="text")
+     * @var string
      */
     private $description;
 
     /**
      * @ORM\Column(type="float")
+     * @var float
      */
     private $price;
 
     /**
-     * @ORM\Column(type="array")
+     * Owning side
+     *
+     * @ORM\ManyToMany(targetEntity="Address")
+     * @ORM\JoinTable(name="courses_addresses",
+     *     joinColumns={@ORM\JoinColumn(name="course_id", referencedColumnName="id", onDelete="CASCADE")},
+     *     inverseJoinColumns={@ORM\JoinColumn(name="address_id", referencedColumnName="id", onDelete="CASCADE")}
```

Comment:
len pre vysvetlenie, to `onDelete` ti zabezpečí, že ak zmažeš kurz alebo adresu, tak sa ti na databázovej úrovni zmaže aj tá väzba... je to možné urobiť aj na úrovni aplikácie (v doctrine), ale takto to úplne stačí

Reply:
ok, díky